gems/sorbet/lib/gem-generator-tracepoint/tracer.rb

```diff
@@ -36,7 +42,15 @@ def extend(mod, *args)
       Object.prepend(ObjectOverride)
 
       module ClassOverride
-        def new(*)
+        # This is a hack due to changes in kwargs with Ruby 2.7 and 3.0. Using
+        # `*` for method delegation is deprecated in Ruby 2.7 and doesn't work
+        # in Ruby 3.0.
+        # See the "compatible delegation" section in this blog post:
+        # https://www.ruby-lang.org/en/news/2019/12/12/separation-of-positional-and-keyword-arguments-in-ruby-3-0/
+        #
+        # Once Sorbet supports exclusively 2.7+ we can remove ruby2_keywords and
+        # use the `...` delegation syntax instead.
+        ruby2_keywords def new(*)
```

Comment:
I suggest that we don't change the `def` but we instead add:
```ruby
send(:ruby2_keywords, :new) if respond_to?(:ruby2_keywords, true)
```
after the method definition.

It is much more surgical and the comment can be added on top of that call.

Reply:
@paracycle that makes sense to me, I've updated the PR to use that pattern. 👍